{
	"header": {
		"line": "CVE-2021-46179 (Reachable Assertion vulnerability in upx before 4.0.0 allows attackers ...)",
		"name": "CVE-2021-46179",
		"description": "(Reachable Assertion vulnerability in upx before 4.0.0 allows attackers ...)"
	},
	"annotations": [
		{
			"line": "\t- upx-ucl 4.2.2-1 (unimportant)",
			"type": "package",
			"package": "upx-ucl",
			"kind": "fixed",
			"version": "4.2.2-1",
			"flags": [
				{
					"severity": "unimportant"
				}
			]
		},
		{
			"line": "\tNOTE: https://github.com/upx/upx/issues/545",
			"type": "NOTE",
			"description": "https://github.com/upx/upx/issues/545"
		},
		{
			"line": "\tNOTE: https://github.com/upx/upx/commit/4a9c46253e308d60b550e9f529e7d37daf978be5 (v3.99)",
			"type": "NOTE",
			"description": "https://github.com/upx/upx/commit/4a9c46253e308d60b550e9f529e7d37daf978be5 (v3.99)"
		},
		{
			"line": "\tNOTE: https://github.com/upx/upx/commit/2d6987252ef4cec9b9051e3e161977ab88b67aac (v3.99)",
			"type": "NOTE",
			"description": "https://github.com/upx/upx/commit/2d6987252ef4cec9b9051e3e161977ab88b67aac (v3.99)"
		},
		{
			"line": "\tNOTE: https://github.com/upx/upx/commit/fcdf0e92c1a2d89188ec7b50fb9b40ef51362560 (v3.99)",
			"type": "NOTE",
			"description": "https://github.com/upx/upx/commit/fcdf0e92c1a2d89188ec7b50fb9b40ef51362560 (v3.99)"
		},
		{
			"line": "\tNOTE: https://github.com/upx/upx/commit/be23f93ee6853a688fb9a920c9ee5222aa212303 (v3.99)",
			"type": "NOTE",
			"description": "https://github.com/upx/upx/commit/be23f93ee6853a688fb9a920c9ee5222aa212303 (v3.99)"
		},
		{
			"line": "\tNOTE: Crash in CLI tool, no security impact",
			"type": "NOTE",
			"description": "Crash in CLI tool, no security impact"
		}
	]
}
